com.unity.netcode.gameobjects/Runtime/Components/Interpolator/BufferedLinearInterpolator.cs

```diff
@@ -220,6 +249,8 @@ public void Reset(T currentValue)
         /// </summary>
         private protected bool m_IsAngularValue;
 
+        private bool m_WasPredictedLerp;
```

Comment:
This doesn't seem to be used

Reply:
Nice catch! 
Thank you for the attention to detail... I need a second set of eyes and keep up the questions... I need everything with this update scrutinized over for sure. 👍